subhub/hub/stripe/customer.py

```diff
@@ -231,15 +263,22 @@ def run(self):
                     messageCreatedAt=int(time.time()),  # required by FxA
                     invoice_id=self.payload.data.object.latest_invoice,
                     customer_id=self.payload.data.object.customer,
-                    created=self.payload.data.object.plan.created,
+                    created=self.payload.data.object.created,
                     plan_amount=self.payload.data.object.plan.amount,
                     eventId=self.payload.id,  # required by FxA
                     canceled_at=self.payload.data.object.canceled_at,
                     cancel_at=self.payload.data.object.cancel_at,
                     cancel_at_period_end=self.payload.data.object.cancel_at_period_end,
+                    currency=self.payload.data.object.plan.currency,
+                    current_period_start=self.payload.data.object.current_period_start,
+                    current_period_end=self.payload.data.object.current_period_end,
+                    invoice_number=invoice_number,
+                    brand=brand,
+                    last4=last4,
+                    charge=charge,
                 )
                 logger.info("customer subscription new recurring", data=data)
-                routes = [StaticRoutes.FIREFOX_ROUTE, StaticRoutes.SALESFORCE_ROUTE]
+                routes = [StaticRoutes.SALESFORCE_ROUTE]
```

Comment:
Was this part of the work here or just an in the neighborhood kind of thing?

Reply:
FxA team asked for it to be removed.

Reply:
Gotcha so that solves the other issue in our tracker that I've seen floating around as part of this.